app/api/common.py

```diff
@@ -14,7 +14,7 @@
 #
 # Returns a SQLAlchemy BaseQuery object. If input state is not None, will return daily data only
 # for the input state.
-def states_daily_query(state=None, preview=False):
+def states_daily_query(state=None, preview=False, limit=None):
```

Comment:
is this new limit logic explicitly tested anywhere?

Reply:
In my previous PR I added tests for fetching single or multiple days, now that the logic has changed, the same tests are still valid (and still pass without changes).